Description: GCC doesn't allow -Wno-format with hardening
  -Werror=format-security .
Author: Chad Miller <chad.miller@canonical.com>

Index: dev.trusty/src/build/common.gypi
===================================================================
--- dev.trusty.orig/src/build/common.gypi	2013-12-04 15:12:48.869650183 -0500
+++ dev.trusty/src/build/common.gypi	2013-12-04 15:12:48.861650718 -0500
@@ -2432,9 +2432,6 @@
               '-Wno-deprecated',
             ],
             'cflags': [
-              # Don't warn about printf format problems.
-              # This is off by default in gcc but on in Ubuntu's gcc(!).
-              '-Wno-format',
             ],
             'cflags_cc!': [
               # TODO(fischman): remove this.
